src/Sarif.Multitool.Library/Rules/SARIF2016.FileUrisShouldBeRelative.cs

```diff
@@ -49,6 +49,40 @@ protected override void Analyze(Run run, string runPointer)
             }
         }
 
+        protected override void Analyze(ArtifactLocation artifactLocation, string artifactLocationPointer)
+        {
+            if (artifactLocation.Uri == null)
+            {
+                return;
+            }
+
+            string uri = artifactLocation.Uri.ToString();
+            if (artifactLocation.Uri.IsAbsoluteUri)
+            {
+                if (artifactLocation.Uri.IsFile)
```

Comment:
>artifactLocation.Uri.IsFile [](start = 20, length = 27)

Fire a note for this case, we observed an absolute file url, which should be rendered instead as a relative URL.